src/components/OnboardingWelcomeVideo.tsx

```diff
@@ -0,0 +1,168 @@
+                    // welcomeVideoStatus === 'animation' it will
+                    // set the same aspect ratio as the video would.
+                    {aspectRatio},
+                ]}
+            >
+                {welcomeVideoStatus === 'video' ? (
+                    <VideoPlayer
+                        url={CONST.WELCOME_VIDEO_URL}
+                        videoPlayerStyle={[styles.onboardingVideoPlayer, {aspectRatio}]}
+                        onVideoLoaded={setAspectRatio}
+                        onPlaybackStatusUpdate={setVideoStatus}
+                        controlsStatus={CONST.VIDEO_PLAYER.CONTROLS_STATUS.VOLUME_ONLY}
+                        shouldUseControlsBottomMargin={false}
+                        shouldPlay
+                        isLooping
+                    />
+                ) : (
+                    <View style={[styles.flex1, styles.alignItemsCenter, {aspectRatio}]}>
+                        <Lottie
+                            source={LottieAnimations.Hands}
+                            style={styles.h100}
+                            autoPlay
+                            loop
+                        />
+                    </View>
+                )}
+            </View>
+        );
+    };
+
+    return (
+        <SafeAreaConsumer>
+            {({safeAreaPaddingBottomStyle}) => (
+                <Modal
+                    isVisible={isModalVisible}
+                    type={shouldUseNarrowLayout ? CONST.MODAL.MODAL_TYPE.CENTERED_UNSWIPEABLE : CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED}
+                    onClose={closeModal}
+                    innerContainerStyle={{
+                        boxShadow: 'none',
+                        borderRadius: 16,
+                        paddingBottom: 20,
+                        paddingTop: shouldUseNarrowLayout ? undefined : MODAL_PADDING,
+                        ...(shouldUseNarrowLayout
+                            ? // Override styles defined by MODAL.MODAL_TYPE.CENTERED_UNSWIPEABLE
+                              // To make it take as little space as possible.
+                              {
+                                  flex: undefined,
+                                  width: 'auto',
+                              }
+                            : {}),
+                    }}
+                >
+                    <View style={[styles.mh100, shouldUseNarrowLayout && styles.welcomeVideoNarrowLayout, safeAreaPaddingBottomStyle]}>
+                        <View style={shouldUseNarrowLayout ? {padding: MODAL_PADDING} : {paddingHorizontal: MODAL_PADDING}}>{getWelcomeVideo()}</View>
+                        <View style={[shouldUseNarrowLayout ? [styles.mt5, styles.mh8] : [styles.mt5, styles.mh5]]}>
+                            <View style={[shouldUseNarrowLayout ? [styles.gap1, styles.mb8] : [styles.mb10]]}>
+                                <Text style={[styles.textHeadlineH1, styles.textXXLarge]}>{translate('onboarding.welcomeVideo.title')}</Text>
```

Comment:
What is the difference between `textHeadlineH1` and `textXXLarge`? What does each of them do here? Sorry for asking so many questions, just trying to get a better grip on our existing styles and looking for ways to consolidate them!

Reply:
Got it, thanks for clarifying. @dubielzyk-expensify @dannymcclain what are your thoughts on that? I see in the Figma file that we're using a slightly larger h1 style for these (28 instead of 22 as pointed out above)... and it does look nice, but I do wonder if it's worth deviating from our standard h1 styles? I don't feel strongly, but always like questioning the exceptions we make.

Reply:
For example, in these sorts of flows, we're just using standard h1 styles:
![CleanShot 2024-04-04 at 09 20 01@2x](https://github.com/Expensify/App/assets/2319350/8f0ce520-d6ae-40db-bad0-036bab2906b3)

Reply:
I've provided two screenshots so you can compare.

Here's the flow with base headline H1 styling + bigger font size override:

<img width="1024" alt="Screenshot 2024-04-04 at 09 50 50" src="https://github.com/Expensify/App/assets/88325488/d8cad27f-bf32-4a42-bd56-5cdaa402e47e">

Here's the flow with the base headline H1 styling:

<img width="1023" alt="Screenshot 2024-04-04 at 09 50 31" src="https://github.com/Expensify/App/assets/88325488/931e3640-cd46-4d81-aeb3-aea287e880a6">

Reply:
Friendly bump @Expensify/design - curious for your thoughts on this one!

Reply:
Personally, I don't think it makes that much of a difference to warrant deviating.

Reply:
>it does look nice, but I do wonder if it's worth deviating from our standard h1 styles?

Sorry, this slipped through the cracks for me somehow. I tend to agree that it does look a teeny bit nicer, but I feel like it's not _meaningfully_ different enough to warrant custom styles/deviation. I think I'd be in favor of just going with our standard H1 style. Curious if Jon might have stronger feelings.